### PR TITLE
Add French and Japanese locales

### LINKED ISSUE #5: Translation
It would be good to have internationalization so non-English speakers could take the test.

I could work in adapting the repository for it; and if the maintainers have a preferred strategy to achieve it, I would follow.

### LINKED ISSUE #39: The Translated (zh-CN) version.
I don't know if it's okey to create pull requests directly to this repo as it seems that there is no i18n support for this project. So I forked and translated a Chinese Simplified version of 8Values [here](https://github.com/LimiQS/LimiQS.github.io). If you think that the translation may be helpful to the project, please feel free to use it. The translation is licensed under CC-BY 4.0 International.
Translated by LimiQS.

### LINKED ISSUE #61: What about other languages?
It would be **really** good to translate the whole webpage to some other languages like Spanish or French.

### PR DESCRIPTION
**Pull request by doamatto**
Extension of #123.

Resolves #61.
Resolves #39.
Resolves #5.

### CODE REVIEW

File: locales/fr/instructions.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+	"title": "Instructions",
+	"description": "Vous allez recevoir une série de décalrations. Pour cacune d'elles, cliquez sur l'option qui correspond le mieux à votre opinion",
```
Comment (yazgazan) [inline locales/fr/instructions.json:3]:
```suggestion
	"description": "Vous allez recevoir une série de déclarations. Pour chacune d'elles, cliquez sur l'option qui correspond le mieux à votre opinion",
```

File: locales/fr/instructions.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+	"confirm": "Je l'ai eu!",
+	"cancel": "Attends, laisse tomber!"
```
Comment (yazgazan) [inline locales/fr/instructions.json:5]: The rest of the French translations are using the formal tone.

```suggestion
	"cancel": "Attendez, laissez tomber!"
```

File: locales/fr/quiz.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+	"strongly_agree": "Tout à fait d'accord",
+	"agree": "D'accord",
+	"neutral": "Neutre/incertain",
+	"disagree": "Pas d'accord",
+	"strongly_disagree": "Pas du totu d'accord",
```
Comment (yazgazan) [inline locales/fr/quiz.json:6]:
```suggestion
	"strongly_disagree": "Pas du tout d'accord",
```

File: locales/fr/translation.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+	"faq_scores": "Je n'aime pas mes résultats !",
+	"faq_scores_answer": "¯\\_(ツ)_/¯<br/>Si vous avez des suggestions ou des critiques constructives, n'hésitez pas à les envoyer à «eightvalues[ at ]gmail.com» ou à <a href=\"https://github.com/8values/8values.github.io/issues/new\">ouvrir un problème sur la page GitHub</a>.",
+
+	"equality": "<b class=\"equality\">Egalité</b> (économique)<br/>Les personnes ayant un score d'égalité élevé pensent que l'économie devrait distribuer la valeur de manière égale parmi la population. Ils ont tendance à soutenir les codes fiscaux progressifs, les programmes sociaux et, à des valeurs élevées, le socialisme.",
+	"wealth": "<b class=\"wealth\">Marchés</b> (Économique)<br/>Les personnes ayant un score de marché élevé pensent que l'économie doit être axée sur une croissance rapide. Ils ont tendance à soutenir la baisse des impôts, la privatisation, la déréglementation et, à des valeurs élevées, le capitalisme de laissez-faire.",
```
Comment (yazgazan) [inline locales/fr/translation.json:13]:
```suggestion
	"wealth": "<b class=\"wealth\">Marchés</b> (Économique)<br/>Les personnes ayant un score de marché élevé pensent que l'économie doit être axée sur une croissance rapide. Ils ont tendance à soutenir la baisse des impôts, la privatisation, la déréglementation et, à des valeurs élevées, le capitalisme de laisser-faire.",
```

File: locales/fr/matches.json [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
+    "econ": ["Communiste","Socialiste","Social","Centriste","Marché","Capitaliste","Laissez-Faire"],
```
Comment (yazgazan) [inline locales/fr/matches.json:2]:
```suggestion
    "econ": ["Communiste","Socialiste","Social","Centriste","Marché","Capitaliste","Laisser-Faire"],
```

**Review by yazgazan — COMMENTED** [review]
I found some typos in the French translations.

Feel free to ignore the ones you disagree with.

File: locales/fr/matches.json (hunk 5 above)
Comment (Genora51) [reply locales/fr/matches.json:2]: Isn't either spelling acceptable when referring to the form of governance?

Comment (yazgazan) [reply locales/fr/matches.json:2]: You're right, `Laissez-Faire` seems to be the correct spelling in this context.

**Comment by Genora51** [comment]
Ok, I'm happy to merge this now

**Pull request merged**